.github/vale/styles/Klaw/branding-warning-temp.yml

```diff
@@ -7,6 +7,4 @@ ignorecase: true
 swap:
   "(?<!Apache )kafka": Apache Kafka
   "(?i)api": API
-  "(?i)kafka connect": Kafka Connect
-  "(?i)klaw core": Klaw Core
-  "(?i)klaw": Klaw
```

Comment:
Removing the rules for "Klaw" and "Kafka Connect" from here should not be part of this PR, could you remove that?

Reply:
Could you also remove the rule for "Kafka Connect"? It's not part of this PR.